docs/pages/setup/guides/joining-nodes-aws.mdx

```diff
@@ -53,14 +93,62 @@ Teleport will use to authenticate. You can provide the IAM credentials to
 teleport through a shared configuration file or environment variables, see
 https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials
 for details.
+  </TabItem>
+</Tabs>
 
 ## Step 2/4. Create the AWS Node Joining Token
 
 Configure your Teleport auth server with a special dynamic token which will
-allow nodes from your AWS account to join your Teleport cluster. Nodes will
-prove that they are running in your AWS account by sending a signed EC2 Instance
-Identity Document which matches an allow rule configured in your AWS Node
-Joining Token.
+allow nodes from your AWS account to join your Teleport cluster.
+
+<Tabs>
+  <TabItem label="IAM Method">
+Under the hood, Nodes will prove that they are running in your AWS account by
+sending a pre-signed `sts:GetCallerIdentity` request to the auth server. The
+Node's identity must match an allow rule configured in your AWS Node Joining
+Token.
+
+Create the following `token.yaml` with an `allow` rule specifying your AWS
+account and the ARN that your Node's identity must match.
+
+```
+# token.yaml
+kind: token
+version: v2
+metadata:
+  # the token name is not a secret because instances must prove that they are
+  # running in your AWS account to use this token
+  name: iam-token
+  # set a long expiry time, the default for tokens is only 30 minutes
+  expires: "3000-01-01T00:00:00Z"
+spec:
+  # use the minimal set of roles required
+  roles: [Node]
+
+  allow:
+  # specify the AWS account which nodes may join from
+  - aws_account: "111111111111"
+  # multiple allow rules are supported
+  - aws_account: "222222222222"
+  # aws_arn is optional and allows you to restrict the IAM role of joining nodes
+  - aws_account: "333333333333"
+    aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-node-role/i-*"
+```
+
+The optional `aws_arn` field in the allow rules supports wildcard characters
```

Comment:
I think we can break up this paragraph a bit to make it clearer with something like:

```markdown
The optional `aws_arn` field in the allow rules supports wildcard characters:

- `*` to match any combination of characters
- `?` to match any single character

See the [IAM docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_resource.html) for more details on this format.
```

Reply:
good suggestion, applied